src/main/java/racingcar/domain/TryCount.java

```diff
@@ -0,0 +1,21 @@
+package racingcar.domain;
+
+public class TryCount {
+    private static final int TRY_COUNT_MIN = 0;
+    private final int tryCount;
```

Comment:
원시값 포장 시 `값의 의미`는 클래스 이름에 표현됐으니 필드는 value 로 표현하는것에 대해서 어떻게 생각하시나요?
토리의 의견이 궁금합니다!

```suggestion
class TryCount {
    ...
    private final int value;

    public int value() {
        return this.value;
    }
}
```

Reply:
처음에는 value보다 값의 의미 자체로 표현하는 게 가독성이 좋다고 생각해 이렇게 사용하게 되었는데, 
밑에 코멘트에도 남겨주신 부분인 getName().getName(), getDistance().getDistance()와 같이 같은 이름의 메서드가 두 번 연속 호출이 되어 저도 어색함을 느끼는 중이었습니다 ㅠㅠ 🥲
메서드를 호출할 때를 생각해보면 찰리께서 추천해주신 방식으로 리팩토링을 하는 것이 더 가독성이 좋을 것 같다고 생각해
TryCount, Name, Distance 클래스의 필드를 모두 value 로 리팩토링하였습니다 ! 😊